JeMPI_Apps/JeMPI_LibMPI/src/main/java/org/jembi/jempi/libmpi/LibMPI.java

```diff
@@ -54,22 +57,34 @@ public LibMPI(
       client = new LibPostgresql(URL, USR, PSW);
    }
 
-   private void sendAuditEvent(
```

Comment:
Can you make the shared as discussed

Reply:
[c8d1dbf](https://github.com/jembi/JeMPI/pull/187/commits/c8d1dbf71d082a82b7c09e89975720483ef626a5)